artifacts/fedora/fedora.go

```diff
@@ -155,16 +168,25 @@ func (f *fedora) releaseMatches(release *Release) bool {
 
 func (f *fedoraGatherer) releaseMatches(release *Release) bool {
 	version, err := strconv.Atoi(release.Version)
-	return err == nil && version >= minimumVersion &&
-		release.Arch == f.Arch &&
-		release.Variant == f.Variant &&
-		strings.HasSuffix(release.Link, "qcow2")
+	if err != nil {
+		return false
```

Comment:
unrelated nit - this isn't really introduced here but returning false after an `err` smells bad to me.

Reply:
I'll take another look.